Assets/Scripts/SS3D/Systems/Tile/Connections/AdjacencyTypes/DirectionnalShapeResolver.cs

```diff
@@ -0,0 +1,225 @@
+﻿using Coimbra;
+using SS3D.Utils;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace SS3D.Systems.Tile.Connections.AdjacencyTypes
+{
+    [Serializable]
+    public struct DirectionnalShapeResolver
```

Comment:
"Directional" is spelt with a single n. Is the extra n in these script names intentional?